quickstart.sh

```diff
@@ -118,6 +227,12 @@ CONTENT_TYPE="Content-Type: application/json"
 # Loop through each JSON file with the prefix "transformed_esci_"
 for file in transformed_esci_*.json; do
     if [[ -f "$file" ]]; then
+
+        if [[ "$file" == "transformed_esci_11.json" && "$full_dataset" == "false" ]]; then
+            echo "Indexing data partially only, exiting loop. Run quickstart.sh with --full-dataset option to index whole dataset."
```

Comment:
"Indexing subset of full data set now, existing loop.  Run..."

Reply:
thanks, that's much better wording, changed!